ts/Core/Series/SeriesDefaults.ts

```diff
@@ -1754,10 +1754,12 @@ const seriesDefaults: SeriesOptions = {
         /**
          * Format for points with the value of null. Works analogously to
          * [format](#plotOptions.series.dataLabels.format). `nullFormat` can
-         * be applied only to series which support displaying null points.
+         * be applied only to series which support displaying null points
+         * i.e `heat map` or `tile map`. Does not work with series that don't
```

Comment:
If putting heatmap and tilemap in code tags (backticks), they must be spelled correctly in one word: `heatmap`, `tilemap`.